FirebaseTestingSupport/Firestore/Tests/QueryFakeTests.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import FirebaseFirestoreTestingSupport
+
+class QueryFakeTests: XCTestCase {
```

Comment:
I assume we would put the tests inside FirebaseCombineSwift/Tests

I've tested this, and it works (of course, we need to set a dependency from the FirebaseCombineSwift pod to the testing support pod),

Reply:
This test is for fake implementation only and may be not 100% required, but it may help in the future debug the test failures and understand if it is the fake implementation issue or its client issue.

Reply:
Yeah, the testing support targets should be added as dependencies to the test targets which is not demonstrated in the current version of the PR.

Reply:
Good point on the docs! Will eventually add them, but feel free to start using it if it unblocks the Combine effort.

Reply:
I added a README file with brief explanations and common steps, PTAL.